src/main/java/me/zeroeightsix/kami/module/modules/player/LagNotifier.kt

```diff
@@ -29,22 +29,25 @@ import net.minecraft.client.gui.GuiChat
         category = Module.Category.PLAYER
 )
 class LagNotifier : Module() {
-    var pauseDuringLag: Setting<Boolean> = register(Settings.b("PauseBaritone", true))
-    private val feedback = register(Settings.booleanBuilder("PauseFeedback").withValue(true).withVisibility { pauseDuringLag.value }.build())
+    private val pauseTakeoff = register(Settings.b("PauseElytraTakeoff", true))
+    var pauseBaritone: Setting<Boolean> = register(Settings.b("PauseBaritone", true))
+    private val feedback = register(Settings.booleanBuilder("PauseFeedback").withValue(true).withVisibility { pauseBaritone.value }.build())
     private val timeout = register(Settings.doubleBuilder().withName("Timeout").withValue(2.0).withMinimum(0.0).withMaximum(10.0).build())
 
     private var serverLastUpdated: Long = 0
     var hasUnpaused = true
+    var takeoffPaused = false
     var text = "Server Not Responding! "
 
     override fun onRender() {
         if ((mc.currentScreen != null && mc.currentScreen !is GuiChat) || mc.isIntegratedServerRunning) return
         if (1000L *  timeout.value.toDouble() > System.currentTimeMillis() - serverLastUpdated) {
-            if (!hasUnpaused && pauseDuringLag.value) {
+            if (!hasUnpaused && pauseBaritone.value) {
                 hasUnpaused = true
                 if (feedback.value) MessageSendHelper.sendBaritoneMessage("Unpaused!")
                 unpause()
             }
+            if (pauseTakeoff.value) takeoffPaused = false
```

Comment:
This shouldn't have the if, because if they change the setting it won't get reset properly